code/web/src/modules/user/api/actions.js

```diff
@@ -14,6 +14,16 @@ export const UPDATE_USER = 'AUTH/UPDATE_USER'
 export const LOGOUT = 'AUTH/LOGOUT'
 
 // Actions
+export function getProducts() {
+  console.log('hi')
+  // return dispatch => {
```

Comment:
Maybe take out these commented lines ?  I guess they are in there because this is supposed to be a working function...